CONTRIBUTING.rst

```diff
@@ -51,33 +51,31 @@ How to Contribute to the Anaconda Installer (the short version)
 
 1) I want to contribute to the upstream Anaconda Installer (used in Fedora):
 
-- base and test your changes on a clone of the ``fedora-<next Fedora number>`` branch.
-- open a pull request for the ``fedora-<next Fedora number>`` branch (``fedora-38``, etc.)
+- base and test your changes on ``main`` branch.
+- open a pull request for the ``main`` branch
 - check the *Commit Messages* section below for how to format your commit messages
 - check the *Release Notes* section below for how to provide a release note
 
 2) I want to contribute to the RHEL Anaconda installer:
 
-- base and test your changes on a clone of the ``rhel-<RHEL number>``  branch.
-- open a pull request for the ``rhel-<RHEL number>``  branch (``rhel-9``, etc.)
+- follow step 1) above to contribute to the ``main`` branch
+- after merging to ``main``, backport the change to the corresponding ``rhel-<RHEL number>`` branch
 - check the *Commits for RHEL Branches* section below for how to format your commit messages
-- check the *Release Notes* section below for how to provide a release note
-
-If you want to contribute a change to both the upstream and RHEL Anaconda then follow both 1) and 2) separately.
```

Comment:
Same here - maybe we can add a note about doing separate PR instead of merging to `main` first only in cases where the Fedora & Rawhide codebases diverged enough to make separate PRs necessary.